### PR TITLE
Add a dedicated category for localization changes

### PR DESCRIPTION
**Pull request by NotMyFault**
The change proposed adds a dedicated section for localization and translation changes to distinguish them better from enhancements or possible other topics you often use for code changes.

Necessary changes to action repositories depending on the emojis are wip.

### CODE REVIEW

**Review by timja — APPROVED** [review]
sounds good to me

File: .github/release-drafter.yml [hunk 1]
```diff
@@ -35,6 +35,9 @@ categories:
       - bugfix
       - regression
       - regression-fix
+  - title: 🌐 Localization and Translation
```
Comment (timja) [inline .github/release-drafter.yml:38]:
```suggestion
  - title: 🌐 Localization and translation
```

or

```suggestion
  - title: 🌐 Localization
```

**Review by timja — APPROVED** [review]

**Comment by NotMyFault** [comment]
Is there something that blocks this PR or we're waiting for I'm unaware of? 👀

**Comment by timja** [comment]
Just to see if any other feedback, given there’s been none let’s merge this

**Pull request merged**